resources/helm/overlays/rls/templates/configmap.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  labels:
+    app: rls
+    istio.io/rev: {{ .Values.revision | default "default" }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+  name: rls-{{ .Values.revision | default "default" }}
+  namespace: {{ .Release.Namespace }}
+data:
+  config.yaml: |
+    #      domain: echo-ratelimit
```

Comment:
these are intentionally commented?

Reply:
maybe we can make this configurable, by rendering e.g. `values.rls.config` into YAML here - that way, users could apply free-form configuration. I'm afraid if we don't offer that, we'll just overwrite the users configuration with every reconcile and this chart will be of little use

Reply:
@shamsher31 yeah, my intention was to provide a simple example - the same found in upstream doc - commented.

@dgn yeah, that's right, this will be overwritten. I think we should use `techPreview.rateLimiting.rawRules` for this, where we get raw yaml from user and apply here. We don't want to wrap this in a API, that's what upstream is working on ATM.